config/rbac/harborconfiguration_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit harborconfigurations.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: harborconfiguration-editor-role
+rules:
+- apiGroups:
+  - harbor-configuration.harbor.configuration
```

Comment:
Generally: Why `harbor-configuration` here and `harborconfiguration` everyhwere else? Seems inconsistent

Reply:
`harbor-configuration` is the api-group name `harborconfiguration` is the resource name, they are different

Reply:
My question is more:
Why did we name the group differently? Nothing else has the `-` in the name AFAICT